docs/maintain/databases.rst

```diff
@@ -167,3 +167,56 @@ To get the table related to a ``pg_toast_*`` table, take the number after ``pg_t
 .. code-block:: sql
 
    SELECT '16712'::regclass;
+
+
+
+Recovering the replica
+----------------------
+
+In the case that replication breaks or the replica server goes offline these are the steps to recover.
+There are two parts to this, mitigating the downtime and fixing replication.
+
+Mitigating downtime:
+
+# Enable public access to postgres on kingfisher-process1
+
+   Modify ``pillar/kingfisher.sls`` setting the following pillar value
```

Comment:
Put this after the code block. We start with generic instructions, then give specific examples.

```suggestion
   For example, for the ``kingfisher-process`` target, modify the ``pillar/kingfisher.sls`` file.
```